index.html

```diff
@@ -1,276 +1,249 @@
-                        <!-- <div class="shape_1"> </div> -->
-                    </div>
-                    <div class="grid_row alternaterow">
-                        <p class="pro_text text_body_font">Multi-Post Stories</p>
-                        <ul class="list_category text_body_font">
-                            <li>Html</li>
-                            <li>CSS</li>
-                            <li>Javascript</li>
-                        </ul>
-                        <a class="project_click_class text_body_font">
-                            <p> See this project</p>
-                            <div class="arrow_key"></div>
-                        </a>
-                    </div>
-
-                    <div class="grid_3 grid_picture mobile_off"></div>
-                    <div class="grid_row mobile_off">
-                        <p class="pro_text text_body_font">Tonic</p>
-                        <ul class="list_category text_body_font">
-                            <li >Html</li>
-                            <li >CSS</li>
-                            <li >Javascript</li>
-                        </ul>
-                        <a class="project_click_class text_body_font">
-                            <p> See this project</p>
-                            <div class="arrow_key"></div>
-                        </a>
-                    </div>
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Emmanuel Portfolio</title>
+
+  <link rel="stylesheet" href="./css/index.css" />
+  <link rel="stylesheet" href="./css/works.css" />
+  <link rel="stylesheet" href="./css/about.css" />
+  <link rel="stylesheet" href="./css/contact.css" />
+  <link rel="stylesheet" href="/css/desktop.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <div class="flex_container">
+    <header>
+      <div class="mobile_logo mobile_off"></div>
+      <nav class="flex_nav text_body_font text_color">
+        <img src="./img/shape-1.svg" class="shape-1" alt="">
+        <img src="./img/shape-2.svg" class="shape-2" alt="">
+        <div class="mobile-icons">
+          <i class="fa fa-bars fa-xl hamb"></i>
+          <i class="fa fa-xmark fa-solid fa-xl close-icon"></i>
+        </div>
+        <img src="./img/img-before.svg" class="img-before" alt="">
+        <ul class="mobile-options">
+          <li class="li_first"><a href="#hello">Hello</a></li>
+          <li class=""><a href="#portfolio">Portfolio</a></li>
+          <li class=""><a href="#about">About</a></li>
+          <li class=""><a href="#contact">Contact</a></li>

```

Comment:
- [ ] Please be sure that the modal is closed and hidden after the user clicks any nav link or the `x` button. Currently, the modal isn't closed or hidden after the user clicks any nav link.
- [ ] Kindly make sure that after the modal is closed through nav links or the `x` button, the page returns to the original design, and only the modal is hidden out of sight. Currently, after the modal is closed, the following behavior occurs. (See image below)

Before opening the modal:
![image](https://user-images.githubusercontent.com/62290545/163207081-b34989c2-9fb4-47a5-89c4-f7e32cb7ec8b.png)
After closing the modal:
![image](https://user-images.githubusercontent.com/62290545/163207292-5d273796-fe98-48fe-9e57-5ba2ff8de4b4.png)